Token.sol

```diff
@@ -34,6 +34,12 @@ contract TokenInterface {
     mapping (address => uint256) balances;
     mapping (address => mapping (address => uint256)) allowed;
 
+    /// Public variables of the token, all used for display 
```

Comment:
This is the constructor: https://github.com/slockit/DAO/blob/master/DAO.sol#L354
The DAO contract derives from TokenCreation ( where the "initial supply" gets created) and Token.sol